shallow_backup/backup.py

```diff
@@ -138,13 +138,22 @@ def run_cmd_if_no_dry_run(command, dest, dry_run) -> int:
 	if not dry_run:
 		overwrite_dir_prompt_if_needed(backup_path, skip)
 
-	for mgr in ["brew", "brew cask", "gem"]:
+	for mgr in ["gem"]:
```

Comment:
This should be refactored. `for x in [item]` is an anti-pattern.

Reply:
True. I was paring this down from the larger PR and it left it like this. This'll be changed with the ruby PR regardless.